packages/vuetify/src/components/VPicker/VPicker.sass

```diff
@@ -12,6 +12,7 @@
   contain: layout style
   display: inline-flex
   flex-direction: column
+  font-size: 1rem
```

Comment:
Should this be a sass variable / pull from a value from the typography sass list?

Reply:
added sass var